readme.md

```diff
@@ -89,6 +89,17 @@ This will list links and logos to all the usergroups.
 
 ### Install locally
 
+### On a Docker Container
+
+```sh
+bundle
+
+docker-compose up postgres -d
+
+./bin/rails db:setup
```

Comment:
Maybe we should follow rails here and use bin/setup. We could even skip the `bundle` step. 
```suggestion
./bin/setup
```

Reply:
yeah, I changed it and removed also bundle from the readme